### PR TITLE
Documentation related to mapviz was added.

### PR DESCRIPTION
**Pull request by samialperen**
Installing from binaries sections were added to the tutorial.

### CODE REVIEW

**Review by wmmc88 — CHANGES_REQUESTED** [review]
change the `mapviz` folder to `uwrt_mapviz` and make it a full package. put the required dependencies in the packages package.xml (mapviz and nmea_navsat). maybe make a launch file launching the required nodes

**Comment by samialperen** [comment]
@wmmc88 Requested changes were done except creating a launch file to call all the required nodes.

File: uwrt_mapviz/package.xml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  <description>Package containing mapviz and nmea_navsat packages to use mapviz with GPS data</description>
+
+  <maintainer email="sami.alperen.akgun@gmail.com">Sami Alperen Akgun</maintainer>
+
+  <license>MIT</license>
+
+  <buildtool_depend>catkin</buildtool_depend>
+
+
+  <build_depend>mapviz</build_depend>
```
Comment (wmmc88) [inline uwrt_mapviz/package.xml]: These are not build depends. they should be exec depends

File: uwrt_mapviz/CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+project(uwrt_mapviz)
+
+## Compile as C++11, supported in ROS Kinetic and newer
+# add_compile_options(-std=c++11)
+
+## Find catkin macros and libraries
+## if COMPONENTS list like find_package(catkin REQUIRED COMPONENTS xyz)
+## is used, also find other catkin packages
+find_package(catkin REQUIRED COMPONENTS
+  mapviz
```
Comment (wmmc88) [inline uwrt_mapviz/CMakeLists.txt]: These aren't needed for build. the only things in this file for "launch/config only" packages should be install directives for the config and launch folders. Here is an example:
https://github.com/ros-simulation/gazebo_ros_demos/tree/kinetic-devel/rrbot_control

File: uwrt_mapviz/README.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
+# uwrt_mapviz
+
+## Installation
+
+## Installing mapviz
+
+### Install from Ros-binaries
+```
+$ sudo apt-get install ros-kinetic-mapviz ros-kinetic-mapviz-plugins ros-kinetic-tile-map
```
Comment (wmmc88) [inline uwrt_mapviz/README.md]: These lines shouldn't be required. The only thing you should typically need to do to install a ros workspace's dependencies should be running `rosdep install --from-paths src --ignore-src -r -y`. It automatically finds all required dependencies based off your package.xml's

File: uwrt_mapviz/README.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+**AT THE END OF WARNING1**
+
+#############################################################
+
+
+## Installing USB GPS Device
+
+### Installing from ros binaries
+```
+$ sudo apt-get install ros-kinetic-nmea-msgs
```
Comment (wmmc88) [inline uwrt_mapviz/README.md]: ditto^

File: uwrt_mapviz/mapviz_configs/uwrobotics1 [hunk 5]
```diff
@@ -0,0 +1,39 @@
+capture_directory: ~
```
Comment (wmmc88) [inline uwrt_mapviz/mapviz_configs/uwrobotics1]: this folder should just be `config`

**Review by wmmc88 — CHANGES_REQUESTED** [review]

File: uwrt_mapviz/README.md [hunk 6]
```diff
@@ -0,0 +1,63 @@
+# uwrt_mapviz
+
+## Installation
+
+## Installing mapviz
+
+#############################################################
+
+**WARNING1:** If you have any error after rosdep install like:
```
Comment (wmmc88) [inline uwrt_mapviz/README.md]: remove this

**Comment by wmmc88** [comment]
@samialperen what's the command to launch mapviz with your configuration?

**Comment by wmmc88** [comment]
@samialperen can you write a shell script or executable python script to check if mapproxy is running in docker yet and if not, start it. You can then write a launch file that brings up both mapproxy and mapviz

**Comment by wmmc88** [comment]
> @samialperen can you write a shell script or executable python script to check if mapproxy is running in docker yet and if not, start it. You can then write a launch file that brings up both mapproxy and mapviz

The shared volume with the configs and cached maps should live somwhere in the `uwrt_mapviz` package... maybe `uwrt_mapviz/mapproxy`. the script should be in `uwrt_mapviz/scripts/`

**Pull request closed without merging**